drawers/cMouseDrawer.cpp

```diff
@@ -167,21 +169,19 @@ void cMouseDrawer::drawToolTipBackground() {
 	allegroDrawer->drawRectangleTransparentFilled(bmp_screen, {shadowX, y + 4, 4, height}, black, 128);
 }
 
-void cMouseDrawer::drawToolTipTurretInformation(cAbstractStructure * theStructure, cTextWriter *textWriter) {
+void cMouseDrawer::drawToolTipTurretInformation(cAbstractStructure * theStructure, cTextWriter& textWriter) {
 	assert(theStructure);
-	assert(textWriter);
 	if (theStructure->belongsTo(player)) {
-		textWriter->writeWithOneInteger("Sight : %d", theStructure->getSight());
-		textWriter->writeWithOneInteger("Range : %d", theStructure->getRange());
+		textWriter.writeWithOneInteger("Sight : %d", theStructure->getSight());
+		textWriter.writeWithOneInteger("Range : %d", theStructure->getRange());
 	} else {
-		textWriter->write("Sight : Unknown");
-		textWriter->write("Range : Unknown");
+		textWriter.write("Range : Unknown");
```

Comment:
Range/Side order got swapped here. (compared to diff & compared to other block in `if`